client/packages/common/src/ui/discovery/DiscoveredServers.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import {
```

Comment:
Relocate to common, see below for chagnes

Reply:
I've added a point in carry over, I can create application package if it should be done now, please let me know